src/pages/Auth/SignUp/index.jsx

```diff
@@ -1,7 +1,50 @@
+import SocialMedias from '@components/Auth/SocialMedias'
+import DataPolicy from '@components/Auth/DataPolicy'
+import { useNavigate } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import routes from '@routing/path'
 import styles from './signup.module.scss'
 
-export default function SignUp() {
+function SignUp(){
+  const navigate = useNavigate()
+  const { t } = useTranslation('nsSignUp')
+
+  function handleNavigate() {
```

Comment:
it's better to rewrite handleNavigate as an arrow function. Here are a few reasons why:
Preserving this context – While not critical in this case since the function doesn’t use this, it can help avoid issues in class components or more complex contexts.
Shorter and more modern syntax, improving readability.
Consistency, we use arrow functions in the project and it's better to maintain a uniform style.